packages/integrations/cloudflare/src/index.ts

```diff
@@ -98,8 +98,28 @@ export type Options = {
 	 * See https://developers.cloudflare.com/kv/concepts/kv-namespaces/ for more details on using KV namespaces.
 	 *
 	 */
-
 	sessionKVBindingName?: string;
+
+	/**
+	 * This configuration option allows you to specify a custom entryPoint for your Cloudflare Worker.
+	 * The entry point is the file that will be executed when your Worker is invoked.
+	 * By default, this is set to `@astrojs/cloudflare/entrypoints/server.js` and `['default']`.
+	 * @docs https://docs.astro.build/en/guides/integrations-guide/cloudflare/#workerEntryPoint
+	 */
+	workerEntryPoint?: {
+		/**
+		 * The path to the entry file. This should be a relative path from the root of your Astro project.
+		 * @example`'src/worker.ts'`
+		 * @docs https://docs.astro.build/en/reference/adapter-reference/#server-entrypoint
+		 */
+		path: string | URL;
+		/**
+		 * The exports to use for the entry file. By default, this is set to `['default']`. If you need to have other top level exports, e.g. DurableObjects you need to provide them here.
+		 * @example `['default', 'MyDurableObject']`
+		 * @docs https://docs.astro.build/en/reference/adapter-reference/#server-entrypoint
+		 */
+		exports: string[];
```

Comment:
Shouldn't this be `exports?:` ? If there's a default, it means that it can be undefined